builds/android/app/src/main/res/values-w820dp/dimens.xml

```diff
@@ -0,0 +1,6 @@
+<resources>
+    <!-- Example customization of dimensions originally defined in res/values/dimens.xml
+         (such as screen margins) for screens with more than 820dp of available width. This
+         would include 7" and 10" devices in landscape (~960dp and ~1280dp respectively). -->
+    <dimen name="activity_horizontal_margin">64dp</dimen>
```

Comment:
You overwrite here the horizontal_margin from 16dp to 64dp for w820dp this could be a hot candidate for the issue.